src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random;
+
+    public ColorSupplier() {
+        random = new Random();
+    }
+
+    public enum Color {
+        RED, BLUE, YELLOW, GREEN, ORANGE, BLACK;
+    }
```

Comment:
If you want write enum here you can better place this in the end and write enum in stack not in line